apps/web/src/server/applications/pages/case/documentation/applications-documentation.controller.js

```diff
@@ -10,9 +16,41 @@ import { getCaseDocumentationCategories } from '../../../lib/services/case.servi
  */
 export async function viewApplicationsCaseDocumentationCategories(request, response) {
 	const { caseId } = response.locals;
-	const documentationCategories = await getCaseDocumentationCategories(caseId);
-
+	const documentationCategories = await getCaseFolders(caseId);
 	const properties = { documentationCategories: sortBy(documentationCategories, ['displayOrder']) };
 
 	response.render(`applications/case/project-documentation`, properties);
 }
+
+/**
+ * View a folder, showing files in the folder, and listing subfolders
+ *
+ * @param {*} request
+ * @param {*} response
+ */
+export async function viewApplicationsCaseDocumentationFolder(request, response) {
+	const { caseId } = response.locals;
+	const { folderId } = request.params;
+	const currentFileCategory = await getCaseFolder(caseId, folderId);
+
+	// get all the sub folders in this folder
+	const subFoldersUnordered = await getCaseFolders(caseId, folderId);
+	const subFolders = sortBy(subFoldersUnordered, ['displayOrder']);
+
+	/** @type { DocumentationPageProps }  */
+	const properties = {
+		caseId,
+		currentFolder: currentFileCategory ?? null,
+		subFolders,
+		folderPath: null,
+		documentationFiles: null
+	};
+
+	// get the folderTree for breadcrumbing
+	properties.folderPath = await getCaseDocumentationFolderPath(caseId, folderId);
```

Comment:
many of these information will be used by every other function in this controller (i.e. by every other page in the documentation section).
would be great to put the common stuff into a application-documentation.locals as like as you did with the pages in the `case` section.

Reply:
I have marked this for TODO in future tickets